content/articles/garbage-collection-in-c#-dotnet-framework/index.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /garbage-collection-in-c#-dotnet-framework/
+title: Garbage Collection in C#.NET Framework
+description: In this article, the reader will be walked through garbage collection in C# Dotnet framework.
+author: elijah-maina
+date: 2021-10-12T00:00:00-15:26
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url:  /engineering-education/garbage-collection-in-c#-dotnet-framework/hero.jpg
+    alt: Garbage collection in c# dotnet framework image
+---
+
+Garbage collection in the C#.Net framework is used to manage Memory automatically. In runtime, heap storage space can be allocated to an object class. If the software has completed all the activities related to the object, the memory location will be a waste since it will not be usable.

```

Comment:
Avoid unneccessary initial capitals such as Memory and Garbage. Again, be consistent.